packages/ui/components/AppLayout/Navbar.tsx

```diff
@@ -0,0 +1,21 @@
+import { Icon } from 'components/Icon';
+
+const Navbar = ({ username }: { username: string }) => {
+  return (
+    <nav className="bg-white py-4 px-8 h-16">
+      <ul className="flex justify-end items-center">
+        <li className="relative mr-8">
+          <Icon name="bell" className="w-6 bell-color" />
+          <div className="absolute -top-1 left-3 w-4 h-4 rounded-full bg-red-500 px-1 py-0 flex flex-col justify-between">
```

Comment:
now i know what you mean by lots of classes!

Reply:
hehehe, ya :D